bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -271,6 +273,9 @@ void PreProcessor::onMessage<ClientPreProcessRequestMsg>(ClientPreProcessRequest
       preProcessorMetrics_.preProcReqIgnored.Get().Inc();
       return;
     }
+    // count requests that would have been ignored if they didn't come from non-primary replicas
```

Comment:
-> Count requests arrived through non-primary replicas and accepted in spite of the same request has been processing by the primary replica